frontend/src/container/SideNav/menuItems.tsx

```diff
@@ -89,6 +90,11 @@ const menuItems: SidebarItem[] = [
 		label: 'Alerts',
 		icon: <BellDot size={16} />,
 	},
+	// {
```

Comment:
Commented code.

Reply:
we need this as we will uncomment it once BE is ready